mmv1/products/workstations/WorkstationConfig.yaml

```diff
@@ -428,6 +428,20 @@ properties:
         description: |
           The service account to use with the specified KMS key.
         required: true
+  - !ruby/object:Api::Type::Array
+    name: 'readinessChecks'
+    description: |
+      Readiness checks to be performed on a workstation.
+    item_type: !ruby/object:Api::Type::NestedObject
+      properties:
+        - !ruby/object:Api::Type::String
+          name: 'path'
+          description: |
+            Path to which the request should be sent.
+        - !ruby/object:Api::Type::Integer
+          name: 'port'
+          description: |
+            Port to which the request should be sent.
```

Comment:
I wonder if we want to mark these two fields required based on the previous error we ran into, even if they seem to be optional based on [the API doc](https://cloud.google.com/workstations/docs/reference/rest/v1beta/projects.locations.workstationClusters.workstationConfigs#readinesscheck). Changing a field from `required` to `optional` is non-changing while the other way around is breaking.

Plus, are these updatable? If so, can you add an update test following https://googlecloudplatform.github.io/magic-modules/develop/test/#add-an-update-test. Otherwise, we need to mark them `immutable`. Also, if you think the fields can be tested in any of the existing tests, feel free to add them to an existing one instead of creating a new test for this feature. Thanks

Reply:
Great suggestion Shuya! I'll make them required, based on the previous error we faced. I would assume they are updatable, I'll introduce a test.

Reply:
Added `readiness_checks` to the basic example, do you think this would be enough? The basic example is part of the existing update test.

Reply:
Would it be possible if we choose another update test? We try to keep only minimum required fields in the basic example config. Sorry about it

Reply:
I don't have an easy way to expose a different port, it's at least not possible via the existing arguments. The default exposed port is port 8080. Could we perhaps omit the update test?